src/helper/request-builder.js

```diff
@@ -99,7 +99,10 @@ RequestBuilder.prototype.setCommonConfiguration = function(ongoingRequest, optio
 
 RequestBuilder.prototype.getTelemetryData = function() {
   var telemetryName = this._universalLoginPage ? 'auth0.js-ulp' : 'auth0.js';
-  var clientInfo = this._telemetryInfo || { name: telemetryName, version: version.raw };
+  var clientInfo = { name: telemetryName, version: version.raw };
+  if (this._telemetryInfo) {
+    clientInfo = objectHelper.extend({}, this._telemetryInfo, { env: clientInfo });
```

Comment:
So this looks like it pushes the standard telemetry info down into the env property... that doesn't sound right to me. @joshcanhelp ?

Reply:
@damieng that's exactly what was wrong 😬